vendor/k8s.io/kubernetes/pkg/proxy/iptables/proxier.go

```diff
@@ -1366,6 +1366,11 @@ func (proxier *Proxier) syncProxyRules(force bool) {
 		return
 	}
 	proxier.changesPending = false
+	// If there are no pending changes, update the healthz server timestamp
+	if proxier.healthzServer != nil {
+		proxier.healthzServer.UpdateTimestamp()
+	}
+	metrics.SyncProxyRulesLastTimestamp.SetToCurrentTime()
```

Comment:
This doesn't seem like it should have any effect; we're already doing exactly this 20 lines further down, and there are no `return`s between here and there...